datafusion/sqllogictest/test_files/map.slt

```diff
@@ -41,14 +68,18 @@ GET 27
 PUT 25
 DELETE 24
 
-query T
-SELECT strings['not_found'] FROM data LIMIT 1;
```

Comment:
can you keep the original test case here since it is non-aggregating? (in contrast to your new query)

Reply:
Great catch. I did not mean to remove the original test. Restored in b3820aafb